spec/src/main/asciidoc/release_notes.asciidoc

```diff
@@ -79,6 +80,29 @@ A full list of changes may be found on the link:https://github.com/eclipse/micro
 === Other changes
 - Clarify failOn() on CircuitBreaker and Fallback (link:https://github.com/eclipse/microprofile-fault-tolerance/issues/240[#240])
 - Circuit Breaker - clarify how requestVolumeThreshold() and rolling window work (link:https://github.com/eclipse/microprofile-fault-tolerance/issues/342[#342])
-- other smaller fixes (link:https://github.com/eclipse/microprofile-fault-tolerance/issues/341[#341]) (link:https://github.com/eclipse/microprofile-fault-tolerance/issues/252[#252]) (link:https://github.com/eclipse/microprofile-fault-tolerance/issues/306[#306]) 
+- other smaller fixes (link:https://github.com/eclipse/microprofile-fault-tolerance/issues/341[#341]) (link:https://github.com/eclipse/microprofile-fault-tolerance/issues/252[#252]) (link:https://github.com/eclipse/microprofile-fault-tolerance/issues/306[#306])
 
+[[release_notes_21]]
+== Release Notes for MicroProfile Fault Tolerance 2.1
 
+This release is a minor release of Fault Tolerance.
+It includes a number of clarifications, which could technically be considered breaking, but in practice shouldn't break anyone.
```

Comment:
I would remove "which could technically be considered breaking, but in practice shouldn't break anyone.", as they don't add much value but might cause confusion.

Reply:
I only included this because I thought we'd do a minor release. There, it does IMHO add quite a lot of value. But I've already seen elsewhere that we plan to do a major release. If so, indeed this needs to be removed.

Reply:
Can you remove the above line as the following sections detail all the changes?

Reply:
Done.